tests/small/test_environ.py

```diff
@@ -117,28 +120,30 @@ def test_getIndexerClustering(default_yml, os_env, output):
 @pytest.mark.parametrize(("default_yml", "os_env", "output"),
             [
                 # Check null parameters
-                ({}, {}, {"label": None, "secret": None, "replication_factor": 1}),
+                ({}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1}),
                 # Check default.yml parameters
-                ({"shc": {}}, {}, {"label": None, "secret": None, "replication_factor": 1}),
-                ({"shc": {"label": None}}, {}, {"label": None, "secret": None, "replication_factor": 1}),
-                ({"shc": {"label": "1234"}}, {}, {"label": "1234", "secret": None, "replication_factor": 1}),
-                ({"shc": {"secret": None}}, {}, {"label": None, "secret": None, "replication_factor": 1}),
-                ({"shc": {"secret": "1234"}}, {}, {"label": None, "secret": "1234", "replication_factor": 1}),
-                ({"shc": {"replication_factor": 0}}, {}, {"label": None, "secret": None, "replication_factor": 0}),
-                ({"shc": {"replication_factor": 1}}, {}, {"label": None, "secret": None, "replication_factor": 1}),
-                ({"shc": {"replication_factor": "2"}}, {}, {"label": None, "secret": None, "replication_factor": 2}),
+                ({"shc": {}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1}),
+                ({"shc": {"label": None}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1}),
+                ({"shc": {"label": "1234"}}, {}, {"pass4SymmKey": None, "label": "1234", "secret": None, "replication_factor": 1}),
+                ({"shc": {"secret": None}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1}),
+                ({"shc": {"secret": "1234"}}, {}, {"pass4SymmKey": "1234", "label": None, "secret": "1234", "replication_factor": 1}),
```

Comment:
Same comments re: test duplication here

Reply:
Nevermind, it's not a duplicate, but let's still add some pass4SymmKey tests.

Reply:
doing just that